applications/FluidDynamicsApplication/python_scripts/trilinos_navier_stokes_solver_vmsmonolithic.py

```diff
@@ -145,10 +171,14 @@ def Initialize(self):
             self.EstimateDeltaTimeUtility = self._GetAutomaticTimeSteppingUtility()
 
         ## Creating the Trilinos convergence criteria
-        self.conv_criteria = KratosTrilinos.TrilinosUPCriteria(self.settings["relative_velocity_tolerance"].GetDouble(),
-                                                               self.settings["absolute_velocity_tolerance"].GetDouble(),
-                                                               self.settings["relative_pressure_tolerance"].GetDouble(),
-                                                               self.settings["absolute_pressure_tolerance"].GetDouble())
+        if (self.settings["time_scheme"].GetString() == "bossak"):
```

Comment:
See comments on openmp version, but choice of scheme should be independent of the choice of convergence criteria.